contributors/design-proposals/storage/volume-topology-scheduling.md

```diff
@@ -79,7 +79,7 @@ pod scheduling.
 
 
 ## Background
-In 1.7, we added alpha support for [local PVs](local-storage-pv) with node affinity.
+In 1.7, we added alpha support for [local PVs](local-storage-pv.pm) with node affinity.
```

Comment:
This is still broken. Did you mean ".md"?

Reply:
oops, fixing.